src/short-css-class-names/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+* Returns an array with valid CSS class names that are between 1 and 3 chars long.
+*
+* Return value in pseudo code:
+* [
+*    "A",
+*    "B",
+*    ...
+*    "Z__",
+* ]
+**/
+module.exports = function() {
+    "use strict";
+
+    let charRange = require("./char-range"),
```

Comment:
I prefer const, since it shows intent. The only case where `let` is OK, even if variable isn't reassigned, is when a object is mutated.